apps/right-to-rent-check/index.js

```diff
@@ -5,13 +5,27 @@ const controllers = require('hof-controllers');
 module.exports = {
   name: 'right-to-rent-check',
   baseUrl: '/right-to-rent-check',
+  params: '/:action?/:id?',
   steps: {
     '/': {
       controller: controllers.start,
```

Comment:
It was a thing when I submitted this PR because hof-bootstrap was not yet at 10.0.0.  

I have sorted this out in a different upcoming PR, if that's OK?

Note: hof-generator created this.  I'll raise it with @lennym to see if he has already captured this otherwise, I'll raise it as an issue